web_widget_open_tab/README.rst

```diff
@@ -0,0 +1,84 @@
+======================
+Widget Open on new Tab
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/11.0/web_widget_open_tab
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-11-0/web-11-0-web_widget_open_tab
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This addon creates a new widget that can be used to open on a new tab a record from a tree.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Edit the tree view and add the widget as the first field, usually, we should use:
+.. code-block:: xml
+
+    <field name="id" widget="open"/>
+
+Now, we can open when clicking with the mouse wheel over the eye.
```

Comment:
When I click with the left mouse button it opens the record in a new tab. When I click with the mouse wheel it opens a new tab not showing the record instead the discuss module.

Reply:
@HviorForgeFlow @tarteo Now this have been fixed.